Generate/src/AllergyIntolerance-3-0/PHR-Client/AllergyIntolerance20/medmij-allergyintolerance-phr-1-2.xml

```diff
@@ -1,30 +1,33 @@
-  <id value="phr-1-2-retrieve-allergyintolerance"/>
-  <version value="stu3-2.0"/>
-  <name value="MedMij AllergyIntolerance - PHR Client - Scenario 1.2 - Retrieve 0 AllergyIntolerance resources"/>
-  <description value="Scenario 1.2 - Retrieve all AlleryIntolerance resources of XXX-TEST-F."/>
-  <nts:patientTokenFixture href="allergyintolerance-nl-core-patient-XXX-TEST-F-token.xml"/>
-  
-  <test id="scenario1-2-retrieve-0-allergyintolerance">
-    <name value="Scenario 1.2"/>
-    <description value="Query all AllergyIntolerance resources of XXX-TEST-F. Response consists of 0 AllergyIntolerance resources."/>
-    <nts:include value="operation-search-phr">
-      <nts:with-parameter name="resource" value="AllergyIntolerance"/>
-    </nts:include>
-    <nts:include value="assert-authorizationHeader" scope="common"/>
-    <nts:include value="assert-request" scope="common"/>
-    <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
-    <nts:include value="assert-responseBundleContent" scope="common"/>
-    <nts:include value="assert-returnCount" scope="project">
-      <nts:with-parameter name="description" value="Confirm that the returned searchset Bundle contains 0 AllergyIntolerance resource."/>
-      <nts:with-parameter name="expression" value="Bundle.entry.where(resource.is(AllergyIntolerance)).count() = 0"/>
-    </nts:include>
-    <nts:include value="assert-returnCount" scope="project">
-      <nts:with-parameter name="description" value="Confirm that the returned searchset Bundle contains 0 entries."/>
-      <nts:with-parameter name="expression" value="Bundle.entry.count() = 0"/>
-    </nts:include>
-  </test>
+    <id value="phr-1-2-retrieve-allergyintolerance"/>
+    <version value="stu3-2.0"/>
+    <name value="MedMij AllergyIntolerance - PHR Client - Scenario 1.2 - Retrieve 0 AllergyIntolerance resources"/>
+    <description value="Scenario 1.2 - Retrieve all AlleryIntolerance resources of XXX-TEST-F."/>
+    <nts:patientTokenFixture href="allergyintolerance-nl-core-patient-XXX-TEST-F-token.xml"/>
+
+    <test id="scenario1-2-retrieve-0-allergyintolerance">
+        <name value="Scenario 1.2"/>
+        <description value="Query all AllergyIntolerance resources of XXX-TEST-F. Response consists of 0 AllergyIntolerance resources."/>
+        <nts:include value="operation-search-phr">
+            <nts:with-parameter name="resource" value="AllergyIntolerance"/>
+        </nts:include>
+        <nts:include value="assert-authorizationHeader" scope="common"/>
+        <nts:include value="assert-request" scope="common"/>
+        <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
+        <nts:include value="assert-responseBundleContent" scope="common"/>
+        <nts:include value="assert-returnCount" scope="common">
+            <nts:with-parameter name="resource" value="AllergyIntolerance"/>
+            <nts:with-parameter name="count" value="0"/>
+        </nts:include>
+        <action>
```

Comment:
Zijn deze twee tests niet dubbelop? Als er geen entries zijn, zullen er ook wel geen AllergyIntolerances zijn. Zou je de check op een lege Bundle ook niet willen veralgemeniseren?

Reply:
Ja klopt. Is wel een inhoudelijke wijziging eigenlijk ;).

Check op een lege bundle zou kunnen, maar komt niet heel veel voor, dus ik laat hem voor nu even links liggen.

Reply:
Ok